index.html

```diff
@@ -37,86 +36,244 @@
+            project you need coded, don’t hestiate to contact me.
+          </p>
         </div>
+        <ul class="container socialGroup">
+          <li class="social-media">
+            <a
+              rel="noopener noreferrer"
+              id="githubButton"
+              href="https://github.com/ssenkooto1"
+              target="_blank"
+              aria-label="GitHub"
+              ><img src="icons/gitHub.svg" alt="Github.com Icon"
+            /></a>
+          </li>
+          <li class="social-media">
+            <a
+              rel="noopener noreferrer"
+              id="linkedinButton"
+              href="www.linkedin.com/in/stephen-ssenkooto-00a036b1"
+              target="_blank"
+              aria-label="LinkedIn"
+              ><img src="icons/linkedin.svg" alt="LinkedIn.com Icon"
+            /></a>
+          </li>
+          <li class="social-media">
+            <a
+              rel="noopener noreferrer"
+              id="angelistButton"
+              href="#"
+              aria-label="Angelist"
+              ><img src="icons/angelist.svg" alt="Angelist.com icon"
+            /></a>
+          </li>
+          <li class="social-media">
+            <a
+              rel="noopener noreferrer"
+              id="twitterButton"
+              href="https://twitter.com/SsenkootoSteph1"
+              target="_blank"
+              aria-label="Twitter"
+              ><img src="icons/twitter.svg" alt="Twitter.com icon"
+            /></a>
+          </li>
+          <li class="social-media">
+            <a
+              rel="noopener noreferrer"
+              id="mediumButton"
+              href="#"
+              aria-label="Medium"
+              ><img src="icons/medium.svg" alt="Medium.com Icon"
+            /></a>
+          </li>
+        </ul>
+      </div>
     </section>
 
-    <section class="contact_se  ction">
-        <div class="contact_paragraph">
-        <p class="paragh_1">
-            I'm always <br>interested in hearing <br>about new projects, so if you'd like to chat please get in touch.
-        </p>
-    </div>
-    <div class="contact_form">
-        <form action="https://formspree.io/f/meqnezez" method="post"> 
-            <input type="text" name="input1" placeholder="Full name" max="30" required class="input1">
-            <hr>
-            <input type="email" name="input2" placeholder="Email address" required class="input2">
-            <hr>  
-            <textarea name="input3" placeholder="Enter text here" cols="20" rows="30" maxlength="500" class="input3" required></textarea>
-            <button type="submit" class="contact_button">Get in touch</button>
-        </form>
-    </div>
+    <section id="projects" class="container works">
+        <div class="container works">
+            <h2> My Recent Works </h2>
+            <span class="line"></span>
+        </div>
+        <div class="gridContainer">
+        </div>
```

Comment:
- [ ] Your projects are supposed to be displayed here but it is currently empty. It would be good to fix that 👍
![f2](https://user-images.githubusercontent.com/85113567/150106673-aeac925e-220c-4647-8a86-aa7fec29f83b.jpg)